apps/src/music/views/TimelineElement.jsx

```diff
@@ -30,8 +31,9 @@ const TimelineElement = ({
   const playerUtils = useContext(PlayerUtilsContext);
   const playingContext = useContext(PlayingContext);
 
+  // TODO: Add length as field on PlaybackEvent to prevent duplicated lookup logic
```

Comment:
Good call.  As discussed separately, this feels like an effect piece of information to cache.